lib/between_meals/repo/git.rb

```diff
@@ -58,7 +58,7 @@ def last_author
       end
 
       def head_parents
-        @repo.head.target.parents
+        @repo.head.target.parents.map {|x| {:time => x.time}}
```

Comment:
I'm not arguing that. I'm saying we have git users who are writing plugins and who are probably assuming stupid crap. You need to at least provide the hash/rev ID.